std_msgs/Header header
common/Uint8TimeStamped cdcu_vehpwrsupply_st
common/Uint8TimeStamped cdcu_chaspwrsupply_st
common/Uint8TimeStamped cdcu_12v1pwrsupply_st
common/Uint8TimeStamped cdcu_12v2pwrsupply_st
common/Uint8TimeStamped cdcu_autdrv24vpwrsupply_st
common/Uint8TimeStamped cdcu_vehpwrswh_st
common/Uint8TimeStamped cdcu_chaspwrswh_st
common/Uint8TimeStamped cdcu_12v1pwrswh_st
common/Uint8TimeStamped cdcu_12v2pwrswh_st
common/Uint8TimeStamped cdcu_autdrv24vpwrswh_st
common/Uint8TimeStamped cdcu_powerstate
common/Uint8TimeStamped cdcu_pwr_prechgst
common/Uint8TimeStamped cdcu_pwr_batlackst
common/Uint8TimeStamped cdcu_pwr_batreadyst
common/Uint8TimeStamped cdcu_pwr_dischgrlyst
common/Uint8TimeStamped cdcu_pwr_chgrlyst
common/Uint8TimeStamped cdcu_pwr_prechgrlyst
common/Uint8TimeStamped cdcu_pwr_shutdowncmd